src/router/index.ts

```diff
@@ -46,6 +46,16 @@ const router = createRouter({
       path: '/tipping',
       name: 'Tippingpage',
       component: () => import('../views/TippingPage.vue')
+    },
+    {
+      path: '/paymentconfirmation',
+      name: 'Payment Confirmation',
+      component: () => import('../views/PaymentConfirmation.vue')
+    },
+    {
+      path: '/vendoroverview',
+      name: 'Vendor Overview',
+      component: () => import('../views/VendorOverview.vue')
```

Comment:
We have `/vendoroverview` and `/vendor/overview`. Can you remove this one again?

Reply:
I removed that part and also removed the redundant file which I forgot earlier. Can you please check if it is alright now? :)